bin/abyss-bbmap

```diff
@@ -0,0 +1,69 @@
+#!/apps/gentoo/bin/bash
```

Comment:
Please change to something that is portable.  Any one of `#!/usr/bin/bash`, `#!bash`, `#!/usr/bin/env bash` are acceptable in my opinion.